documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -0,0 +1,12 @@
+org.infinispan.client.hotrod.configuration.ConfigurationBuilder cb
+      = new org.infinispan.client.hotrod.configuration.ConfigurationBuilder();
+cb.tcpNoDelay(true)
+  .connectionPool()
+      .numTestsPerEvictionRun(3)
+      .testOnBorrow(false)
+      .testOnReturn(false)
+      .testWhileIdle(true)
```

Comment:
This is maybe a question for @tristantarrant, but are the connection pool params often changed? Maybe a better example would be defining a custom marshaller, or statistics?

Reply:
@oraNod Could you make this change? Then I'll merge

Reply:
@ryanemerson could you please take another look. Tristan provided an example for hotrod-client.properties. maybe we should make the `ConfigurationBuilder()` examples match the properties file? 

also, I'm not sure about the statistics examples specifically the value for .jmxName()